app/services/ccms/attribute_configuration.rb

```diff
@@ -0,0 +1,24 @@
+module CCMS
+  class AttributeConfiguration
+    VALID_APPLICATION_TYPES = %i[standard non_passported].freeze
+
+    attr_reader :config
+
+    # Expects application type of :standard or :non_passported
+    def initialize(application_type:)
+      raise ArgumentError, 'Invalid application type' unless application_type.in?(VALID_APPLICATION_TYPES)
+
+      @config = YAML.load_file(configuration_files[:standard])
+      @config.deep_merge! YAML.load_file(configuration_files[application_type]) if application_type != :standard
```

Comment:
is the `if application_type != :standard` required? since it's being checked in the raise error above `unless application_type.in?(VALID_APPLICATION_TYPES)`